packages/custom-functions-metadata/Specification.md

```diff
@@ -195,3 +196,23 @@ A function can return a Promise, which will provide the value when the promise i
 ### Other types
 
 Any other type will be treated as an error.
+
+---
+## Custom enums
+
+The `@customEnum` JSDoc tag can be used in TypeScript to mark an enum as a custom enum. When a custom function uses this custom enum as the type for its arguments, the enum values will be displayed in a dropdown list for the end user when they enter the function.
+
+### Tag @customEnum
+
+Syntaxt: @customEnum _{type}_
```

Comment:
If the type is optional, this can be represented with square brackets, which is a common way to do it.

Reply:
This spec file seems not having a fixed practice to denote optional, but _italic_ is used in most cases, for example `@customfunction _id_ _name_`. 
Do you think we should update them all to use square brackets?